### ISSUE TITLE
[Bug Bash][Test Failure][Accessibility]Hotkey “Del” for package context menu “Remove” in Solution Explorer doesn’t work 

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#8860]
## Details about Problem
NuGet version: NuGet Client Dev\5.5.0.6326
VS Version: Master\29526.122
OS: Windows-10-Enterprise-RS5
## Detailed repro steps 
1. Create a project [e.g. C# Console App (.NET Core 3.1) template] via VS.
2. Open PM UI of the project by right-clicking the project->Manage NuGet Packages… in Solution Explorer.
3. Install several packages.
4. Open Solution Explorer window and go to Dependencies->Packages node.
5. Click an installed package and press the hotkey “Del” on keyboard. (The hotkey is described on the context menu of the installed package.)
## Expected
The package should be deleted successfully. (Same with clicking the context menu: “X Remove”)
## Actual
Hotkey “Del” for package context menu “Remove” in Solution Explorer doesn’t work as below screenshot.
![HotkeyDelDoesn'tWork](https://user-images.githubusercontent.com/20632625/69719018-9697bc00-114a-11ea-82fb-3398aacebaa3.png)
## Note
1. The package can be deleted successfully after pressing the context menu: “X Remove”.
2. The hotkey “Alt+Enter” for package context menu “Properties” works well.
3. It repro on 16.1 GA (D16.1\28916.193) with implicit NuGet 5.1.0.6013.

**Comment by aortiz-msft** [comment]
I wasn't able to repro this issue. Pressing Del while a Package is selected brings up the uninstall dialog. Note that the hotkey will not work while the context menu is up.
